### ISSUE TITLE
Context menu on Mac?

### ISSUE THREAD

**Issue by AndrewCapon** [greatscottgadgets/packetry#216]
Hi Guys,

I noticed there was a new release with saving of data via a context menu.

Is this meant to be a right click menu as I am getting nothing on the Mac?

Cheers

Andy

**Comment by miek** [comment]
Yeah, there should be a right click menu, though it will only show up when right clicking on items that have a way to save data.
Currently that's control/data transfers, transactions, and packets that have a payload, and also descriptors.

**Comment by AndrewCapon** [comment]
Thanks, I'll test with those...
